app/src/androidTest/java/com/android/streetworkapp/ui/Progression/ProgressionTest.kt

```diff
@@ -84,7 +85,7 @@ class ProgressionTest {
     composeTestRule.onNodeWithTag("percentageInsideCircularProgressBar").assertIsDisplayed()
     composeTestRule.onNodeWithTag("scoreTextUnderCircularProgressBar").assertIsDisplayed()
     composeTestRule.onNodeWithTag("metricCardScoreValue").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("metricCardParksVisitedValue").assertIsDisplayed()
+    // composeTestRule.onNodeWithTag("metricCardParksVisitedValue").assertIsDisplayed()
```

Comment:
comment: if this assert will not be used anymore, might as well delete it

Reply:
Done, thank you!